sycl/tools/sycl-vxx/bin/sycl_vxx.py

```diff
@@ -556,6 +556,9 @@ def _link_kernels(self, kernels):
                 if has_assignment and has_default:
                     raise NotImplementedError(
                         "Mix between assigned an non assigned bank is not supported yet")
+        for pipe in self.kernel_properties['pipe_connections']:
```

Comment:
Add a comment like `The pipe plumbing is actually done by Vitis with the right options`